tests/pg_regress/sql/explain.sql

```diff
@@ -12,8 +12,19 @@ SELECT * FROM test_explain;
 -- Allows query execution as it is an EXPLAIN ANALYZE, as it may be useful to understand the plan
 EXPLAIN (ANALYZE, COSTS OFF, TIMING OFF, SUMMARY OFF, BUFFERS OFF) SELECT * FROM test_explain;
 
+-- Allows query execution as it is an EXPLAIN ANALYZE, even for parallel queries
+SET parallel_setup_cost = 0;
+SET parallel_tuple_cost = 0.000001;
+CREATE TABLE test_explain_parallel (id, migrated_at) AS (SELECT generate_series(1, 800000)::bigint id, null::timestamp without time zone);
```

Comment:
Could we maybe have something like this?
```suggestion
CREATE TABLE test_explain_parallel (id, migrated_at) AS (SELECT generate_series(1, 800000)::bigint id, null::text);
```

Because the timestamp without time zone looks a bit weird to keep for ever ;)
I know the example comes from me ;)

Reply:
Haha, sure!